website/src/components/EventSlider/style.ts

```diff
@@ -33,16 +33,17 @@ const useStyles = makeStyles((theme) => ({
     margin: theme.spacing(2, 0, 2, 2),
     "& div.slick-active": {
       "& + .slick-active": {
-        "& + .slick-slide": {
+        "& + .slick-slide:not(:last-child)": {
           "&  > div": {
             opacity: ".3",
+            pointerEvents: 'none'
           },
         },
       },
     },
     [theme.breakpoints.down("sm")]: {
       "& div.slick-active": {
-        "& + .slick-active": {
+        "& + .slick-active:not(:last-child)": {
```

Comment:
This is to avoid giving opacity to the last child when there is only enough slides as slides to show.